### PR TITLE
rogreen/add deployment

### PR DESCRIPTION
**Pull request by georgettica**
- feat(route-monitor-operator): add deployment
- feat(route-monitor-operator): add future releases

### CODE REVIEW

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@ images:
 - dockerfile_path: Dockerfile
   from: base
   to: route-monitor-operator
+promotion:
+  name: "4.7"
```
Comment (georgettica) [inline ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:16]: should this be 4.8?

**Comment by georgettica** [comment]
version tagging issues, fixing now

**Comment by georgettica** [comment]
currently not sure how to proceed

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml (hunk 1 above)
Comment (cblecker) [reply ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:16]: This promotion section isn't needed at all, and should be removed.

**Review by georgettica — COMMENTED** [review]
I am closing this PR after discussion and finding a better place to do it, if needed will reopen﻿

**Pull request closed without merging**